azkaban-exec-server/src/test/java/azkaban/execapp/JobRunnerTest.java

```diff
@@ -143,7 +140,7 @@ public void testDisabledRun() {
     final MockExecutorLoader loader = new MockExecutorLoader();
     final EventCollectorListener eventCollector = new EventCollectorListener();
     final JobRunner runner =
-        createJobRunner(1, "testJob", 1, false, loader, eventCollector);
+        createJobRunner(1, "testJob", 0, false, loader, eventCollector);
```

Comment:
This change is in already enabled test `testDisabledRun`: no reason to have sleep seconds=1 as the job is disabled and skipped immediately.